core/src/main/java/com/microsoft/applicationinsights/internal/quickpulse/DefaultQuickPulseDataFetcher.java

```diff
@@ -75,7 +75,7 @@ private DefaultQuickPulseDataFetcher(ArrayBlockingQueue<HttpPost> sendQueue, Tel
         formatDocuments(sb);
         sb.append("\"Instance\": \"").append(instanceName).append("\",");
         sb.append("\"InstrumentationKey\": \"").append(ikey).append("\",");
-        sb.append("\"InvariantVersion\": 1,");
```

Comment:
@trask Do you know why this value and the one below in `DefaultQuickPulsePingSender.java ` are different? They really should not be, as far as I understand.

Reply:
hey @rambhatt-msft, I'm not sure why, it does seem like a mistake that they are different. I looked at the git history and found this: #904. Is it ok for us to go to v2/v3, even though we don't support filtering and sample telemetry?

Reply:
We should keep the version unchanged at 1. Since higher versions, like you said, support filtering and sample telemetry and the version numbers are used to distinguish between them. This change I did now is gonna be in all SDKs so its not a distinguishing feature.